### ISSUE TITLE
Multiple Input Streams

### ISSUE THREAD

**Issue by GoatedChopin** [GoatedChopin/XRayBot#3]
[](https://www.youtube.com/watch?v=4-O14gOdRso)

This video is a great tutorial on how to build a keras model with mixed_data, which could easily be adapted to this use-case. Although his pokemon health-regressor model predicts a continuous variable, **changing the output layer** should be sufficient alteration.

**Comment by GoatedChopin** [comment]
Implement checkpoints that save the best model using tensorflow.keras.callbacks.ModelCheckpoint

found at 45:10 in Greg Hogg's video. That should save on computation time if a certain number of epochs is best.

**Comment by GoatedChopin** [comment]
change final activation layer to softmax if not already
